src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0px;
+}
+
+[class*="item-"] {
```

Comment:
it's good but I recommend to use class. It will be easier to read and debug code.